src/typings/Types.ts

```diff
@@ -220,8 +221,11 @@ export interface RangeTarget {
   active: PrimitiveTarget;
   excludeAnchor: boolean;
   excludeActive: boolean;
+  rangeType?: RangeType;
 }
 
+export type RangeType = "vertical";
```

Comment:
as mentioned above, I'd be tempted to add another option and then make it required in the full `RangeTarget`